ppr-ui/src/composables/mhrInformation/useMhrInformation.ts

```diff
@@ -114,6 +115,9 @@ export const useMhrInformation = () => {
     const currentOwnerGroups = data?.ownerGroups || [] // Safety check. Should always have ownerGroups
     await parseMhrHomeOwners(cloneDeep(currentOwnerGroups))
 
+    const unitNotes = data?.notes || []
+    await setMhrUnitNotes(unitNotes)
```

Comment:
Okkk, now I see what you are doing here. You just parsing the notes from registrations. 👍🏻

Reply:
This is where we could use `mockUnitNotes` until we can get some dummy data in dev.